test/e2e/multi-stage/cluster-claim.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  cli:
+    name: "4.8"
+    namespace: ocp
+    tag: cli
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+resources:
+  '*':
+    limits:
+      cpu: 500m
+    requests:
+      cpu: 10m
+tests:
+  - as: e2e-claim
+    cluster_claim:
+      architecture: amd64
+      cloud: aws
+      owner: fake
+      product: ocp
+      timeout: 1h0m0s
+      version: "4.7"
+    steps:
+      test:
+        - as: claim-step
+          commands: |
```

Comment:
Which of these commands will fail if the claim does not work? Why are all of the commands necessary?

Reply:
I have to see what hive gives us in the secret and just my cmd accordingly.